### DISCUSSION TITLE
Participation in other open source events

### DISCUSSION THREAD

**Discussion by hexsecs** [uptane/uptane.github.io#129] [General]
Hacktoberfest was a great success. It would be great if we had a plan for events to participate in throughout the year. @Abhijay007 , do you have suggestions for other events we should consider?

**Comment by Abhijay007** [comment]
I will surely share a few more programs soon, thanks for starting this discussion

**Reply by Abhijay007** [reply]
Hi @hexsecs I think we can apply for this one : https://24pullrequests.com/

**Reply by Abhijay007** [reply]
and can list our community here too : https://aviyel.com/

**Reply by ashmitjsg** [reply]
Hi @hexsecs, what are your thoughts on GSoC or LFX Mentorship Program?

**Reply by hexsecs** [reply]
@devilkiller-ag - GSoC is great! We participated this summer and @Abhijay007 did a fantastic job with the site.

**Reply by ashmitjsg** [reply]
@Abhijay007 I have tried 24pullprequests before. I would love to contribute to uptane through it.

**Reply by ashmitjsg** [reply]
Great @hexsecs, will we be participating in 2024  too?

**Reply by Abhijay007** [reply]
> Hi @hexsecs, what are your thoughts on GSoC or LFX Mentorship Program?

Hi @devilkiller-ag, GSoC and LFX are great open-source programs indeed, and we are certainly interested in participating in them if we come across compelling project ideas. Currently, we are primarily focused on engaging with open-source events, (such as Hacktoberfest, 24PullRequests, etc) and expanding our community outreach efforts to make more potential contributors aware of Uptane and its projects. If we decide to participate in GSoC 2024, LFX, or any other open-source program, we will certainly notify our contributors about the opportunities.

**Reply by Abhijay007** [reply]
> @Abhijay007 I have tried 24pullprequests before. I would love to contribute to uptane through it.

@devilkiller-ag Since you have been a past contributor at 24pullrequest, it would be greatly appreciated if you could provide guidance on the same. Your insights are valuable to us. Could you please advise us on the process of applying as an organization in 24pullrequest?

**Reply by ashmitjsg** [reply]
Sure @Abhijay007, We can register our project on 24PullRequests at here: https://24pullrequests.com/projects/new. I encourage to register it as a project owner.

**Comment by Abhijay007** [comment]
> Sure @Abhijay007, We can register our project on 24PullRequests at here: https://24pullrequests.com/projects/new. I encourage to register it as a project owner.

Thanks for sharing this we will surely register for the same soon, cc : @hexsecs

**Comment by Abhijay007** [comment]
@hexsecs Uptane is now part of [24pullrequests](https://24pullrequests.com/) program, you can follow this discussion here : https://github.com/24pullrequests/24pullrequests/discussions/4119 , I think it will be great if we create an announcement post around the same both on LinkedIn and Update site, we will share the announcement post afterward on discord too with the respected links, I think it would be great if we do it in the upcoming week only as that way we get some time to open new issues and find volunteers for the same.

**Comment by Abhijay007** [comment]
Organization/Community applications for [Outreachy](https://www.outreachy.org/) are currently open. I believe our participation in this program could be valuable. To assist in understanding the eligibility criteria and application procedures, I have compiled supporting documents. You can find details on eligibility criteria and specific procedures for organizations and communities on the Outreachy documentation page: https://www.outreachy.org/docs/community/. A table of contents is available on this page, and you can specifically read about organization/community eligibility [here](https://www.outreachy.org/docs/community/#participation-rules).

Outreachy provides funding for interns from its general fund, but there is a eligibility criteria toapply and access these fund. To learn more about the funding requirements, please visit https://www.outreachy.org/docs/community/#funding-requirements. If we meet the eligibility criteria, I strongly recommend considering participation in this program. The application deadline for organization submissions is around February 16. Additional information about the application process can be found [here](https://www.outreachy.org/communities/cfp/).

cc @hexsecs , @jhdalek55 , @iramcdonald – could you please review this information and consider our potential involvement in Outreachy?
